prettier.config.js

```diff
@@ -0,0 +1,25 @@
+// Add to .prettierignore to ignore files and folders
+
+// This configuration all the formats including typescript, javascript, json, yaml, markdown
+module.exports = {
+  tabWidth: 2,
+  printWidth: 120,
+  semi: false,
+  singleQuote: false,
```

Comment:
To reduce the code diff we can modify these, but that means changing the preset that the organization uses.

Reply:
I would say use organization preset. I would rather have consistency than smaller diff.

Reply:
OK. I will revert this then.